<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquipmentDetail</class>
 <widget class="QMainWindow" name="EquipmentDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>مشخصات دستگاه</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>مشخصات دستگاه</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="4">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="equip_typeEdit">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>نوع دستگاه:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="equipment_periority">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>وضعیت کلی</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="equip_locEdit">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>محل استقرار:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="5" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="equip_codeEdit">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>کد دستگاه:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="8">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>PM Checklist</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QTableWidget" name="pmTable">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="columnCount">
             <number>5</number>
            </property>
            <column>
             <property name="text">
              <string>وضعیت</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>موعد بعدی</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>آخرین انجام</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>تناوب</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>New Column</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>812</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="doneBtn">
               <property name="text">
                <string>DONE</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="editBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Edit</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="codeLabel"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
